data/schema.ttl

```diff
@@ -8606,7 +8606,7 @@ While such policies are most typically expressed in natural language, sometimes
     rdfs:label "skills" ;
     rdfs:comment "A statement of knowledge, skill, ability, task or any other assertion expressing a competency that is desired or required to fulfill this role or to work in this occupation." ;
     :domainIncludes :JobPosting,
-        :Occupation ;
+        :Occupation, :Person ;
```

Comment:
the issue mentions this can only apply to an Organization, should that be added as well?

Reply:
Yes, updated. PTAL.